clients/drcachesim/reader/config_reader.cpp

```diff
@@ -145,6 +145,18 @@ config_reader_t::configure(std::istream *config_file, cache_simulator_knobs_t &k
             } else {
                 knobs.model_coherence = false;
             }
+        } else if (param == "use_physical") {
```

Comment:
This looks unrelated: please separate into a new PR.

Reply:
._.

Reply:
Done.